jep/0000/README.adoc

```diff
@@ -0,0 +1,358 @@
+Incompatibilities will be reported and tracked as plugin issue reports.
+
+=== New policy: Jenkins core with Java 11
+
+The following policy is suggested:
+
+* Jenkins core components will be compiled with Java 11 and will require Java 11 or later at runtime
+* Jenkins plugins that depend on a Jenkins core that requires Java 11 must be compiled with Java 11
+** In order to support releases that only run with Java 11, the plugins must use the plugin POM that adds support for `java.level` 11
+
+This policy may require patches in parent POMs:
+
+* Parent POMs will be updated as needed, including
+  link:https://github.com/jenkinsci/plugin-pom[Plugin POM],
+  link:https://github.com/jenkinsci/pom[Jenkins POM], and
+  link:https://github.com/jenkinsci/bom[Jenkins plugin bill of materials POM],
+
+=== Rollout plan
+
+The rollout procedure should be coordinated within the link:https://jenkins.io/sigs/platform/[Platform SIG].
+
+==== Timeline (TBD)
+
+* Announce Java 8 end of support for weekly in blogpost
+** Describe Java 11 upgrade process for users of war, deb, rpm, and msi installations
+* Add higher visibilty warning on Java 8 end of support in Jenkins weekly (an idea that is not yet fully formed)
+* Remove Java 8 from weekly core release, weekly Docker controller images
+* Announce Java 8 end of support for LTS in blogpost
+* Remove Java 8 from LTS core release, include in changelog and upgrade guide
+* Remove Java 8 from Docker agent images
+
+==== Website
+
+* link:https://jenkins.io/doc/administration/requirements/java/[Java Support Page] is updated to state the weekly version of Jenkins core and the LTS version of Jenkins core that last support Java 8
+* A blogpost is provided that announces the change in weekly releases and outlines the steps administrators must take to make the change
+** War file installations
+** Docker installations
+** MSI installations on Windows
+** RPM and DEB installations on Linux
+* A blogpost is provided that announces the change in an LTS release and outlines the steps administrators must take to make the change
+* The LTS changelog and upgrade guide describes the steps administrators must take to make the change
+* A webinar is presented that outlines the changes and outlines the steps administrators must take to make the change
+
+==== Issue tracking
+
+* Issues related to Java 8 end of support are tracked as Jenkins issues
+** link:https://issues.jenkins.io/browse/JENKINS-67688[JENKINS-67688] is the Jira epic that tracks issues in Jira
+** Plugins that use GitHub issues will place a link to their GitHub issue into the Jira epic
+
+==== Post-release support
+
+After the end of Java 8 support in the weekly releases, there may be a number of issues reported by early adopters.
+Core maintainers will respond to issue reports as they did for configuration form modernization ("table to div").
+A Jira label `java8-end-of-support` will be assigned to issue reports related to Java 8 end of support.
```

Comment:
We already have a jumble of labels all essentially about the same thing. I think this wording is wrong. There is no issue about ending Java 8 support per se, or at least none we intend to do anything about; issues would be that something is broken in Java 11, for which we already have labels, and that this has now become critical because it is impossible to update Jenkins core until the issue is addressed or worked around.